src/commands.rs

```diff
@@ -1558,8 +1558,9 @@ fn log_template(settings: &UserSettings) -> String {
     };
     let default_template = format!(
         r#"
-            change_id.short()
-            if(divergent, label("divergent", " divergent"))
+            if(divergent,
+              label("divergent", change_id.short() "??"),
+              change_id.short())
```

Comment:
I have been working on the some changes to the templater and formatter and added #992 with a simple test of some of the current colors. I think it would be good to have a similar test for divergent changes. What do you think?

Reply:
Sure, this should be really easy given a previous test I added.